src/codeu/chat/common/ServerInfo.java

```diff
@@ -15,8 +15,7 @@ public ServerInfo() {
         try {
             this.version = Uuid.parse(SERVER_VERSION);
         }
-        catch(IOException e)
-        {
+        catch(IOException e) {
```

Comment:
nit: move the catch statement to the line above so that it looks like `} catch(...`